### DISCUSSION TITLE
Alpha Build v0.0.4 - Nyxtext Zenith - Windows only

### DISCUSSION THREAD

**Discussion by parazeeknova** [parazeeknova/nyxtext-zenith#6] [Announcements]
<h3 align="center">
	<img src="https://github.com/parazeeknova/nyxtext-zenith/blob/main/.github/assets/Zenith-banner.png?raw=true" alt="Bannner"/>
<br/>

### Alpha v0.0.3 - Nyxtext Zenith - Standalone (Windows Only) release

> [!NOTE]
> Because the application is not signed by Microsoft, Windows Defender blocks it; add an exclusion in Defender

### Changelog :
- The statusbar now displays the current cursor line, column, total words, lines, and the lexer used in the codespace.
- Codespace now supports over 35+ languages. (Detected automatically based on the file extension)
- Catppuccin customized lexer for `Python`, `Javascript`, `C++`, `HTML`, `CSS`, `Markdown`, `Lua`, `JSON`. More to come soon.
-  Lexer's can be customized using lua in `zenith/color_schemes`.
- Added a build script for Windows that allows you to build from scratch.
- File cache up to ten files.
- Thread Optimizations for `zenith-core`.

### Installation :
- Extract and run `Nyxtext.exe`

### CLI Usage: 
To open Files: `Nyxtext.exe -o <filename>` or `python -m zenith -o <filename>`

**Full Changelog**: https://github.com/parazeeknova/nyxtext-zenith/compare/alpha-v0.0.3...alpha-v0.0.4

<hr /><em>This discussion was created from the release <a href='https://github.com/parazeeknova/nyxtext-zenith/releases/tag/alpha-v0.0.4'>Alpha Build v0.0.4 - Nyxtext Zenith - Windows only</a>.</em>

**Comment by parazeeknova** [comment]
@rohankishore So I have finally decided to shift from TKinter to Pyside and then Pyqt (which I should have done sooner). Could you please take a look and suggest any changes or ideas? Given your knowledge of PyQT6, if you have a moment, could you kindly review it ..... 

- :)

**Comment by rohankishore** [comment]
Yep. I'll review and try to contribute to it. 

Thanks.

**Reply by parazeeknova** [reply]
I look forward to it :)

**Comment by rohankishore** [comment]
Heya! 

Sorry for the late reply. I've checked it out and it's really good for a first edition editor. I also would like to know about how you integrated powershell btw.
